spring-cloud-huawei-swagger/src/main/java/com/huaweicloud/swagger/ServiceCombSwaggerHandlerImpl.java

```diff
@@ -58,19 +61,18 @@ public void setOpenApiResource(OpenApiResourceWrapper openApiResource) {
 
   @Override
   public void init(String appName, String serviceName) {
-    runSpringDocScanner();
+    runSpringDocScanner(serviceName);
   }
 
-  private void runSpringDocScanner() {
+  private void runSpringDocScanner(String serviceName) {
     SpringMvcOpenApiResource mvcOpenApiResource = openApiResource.createOpenApiResource(Constants.DEFAULT_GROUP_NAME);
     Set<String> set = mvcOpenApiResource.getControllers();
     set.forEach(key -> {
       SpringMvcOpenApiResource beanOpenApiResource = openApiResource.createOpenApiResource(key);
       beanOpenApiResource.clearCache();
-      swaggerMap.put(key, beanOpenApiResource.getOpenAPI());
+      openAPIList.add(beanOpenApiResource.getOpenAPI());
     });
```

Comment:
springdoc has already create one resource, need to separated and merged it again

Reply:
fixed